pkg/apis/operator/v1alpha1/elevator_types.go

```diff
@@ -211,6 +219,61 @@ func (s *Elevator) SetUnReadyCondition() bool {
 	return false
 }
 
+// IsPaused returns if the Elevator is paused.
+func (s *Elevator) IsPaused() bool {
+	if s.Generation != s.Status.ObservedGeneration {
+		return false
+	}
+
+	for _, condition := range s.Status.Conditions {
+		if condition.Type == ElevatorPaused &&
+			condition.Status == ConditionTrue {
+			return true
+		}
+	}
+	return false
+}
+
+func (s *Elevator) SetPausedCondition() bool {
```

Comment:
```suggestion
func (s *Elevator) Pause() bool {
```

Reply:
let's leave it consistent with `SetReadyCondition` and `SetTerminatingCondition`